core/eolearn/core/utils/common.py

```diff
@@ -102,6 +102,12 @@ def _apply_to_spatial_axes(
     After achieving 2D data slices the mapping function is applied. The data is then reconstructed into original form.
     """
 
+    if len(spatial_axes) != 2:
+        raise ValueError("Parameter `spatial_axes` should be tuple of two increasing int smaller then `data.ndim`")
```

Comment:
not really required IMO, the type is specific enough

Reply:
Removed.